code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,25 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/image/IM
```

Comment:
Shouldn't be an image. We only use `mutable_appearance` now.

Reply:
Fixed